packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -187,6 +188,27 @@ export class Meter implements types.Meter {
     this._metrics.set(name, metric);
   }
 
+  /**
+   * Provide a pre-computed re-useable LabelSet by
+   * converting the unordered labels into a canonicalized
+   * set of lables with an unique identifier, useful for pre-aggregation.
+   * @param labels user provided unordered Labels.
+   */
+  labels(labels: types.Labels): types.LabelSet {
+    let keys = Object.keys(labels).sort();
```

Comment:
```suggestion
    const keys = Object.keys(labels).sort();
```

Same for `identifier` and `sortedLabels`.